apps/info.css

```diff
@@ -1,10 +1,16 @@
 html,
 body {
-			width: 100%;
-      height: 100%;
-      min-height: 100%;
-			display: flex;
-			flex-direction: column;
+	width: 100%;
+	height: 100%;
+	min-height: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+footer{
+  position: absolute;
```

Comment:
Having an issue with this; it does not stay at the bottom of the screen like I presume you intend, but when scrolling down, it stays fixed to the document. That is:
![image](https://user-images.githubusercontent.com/4951803/104249513-98e5d400-5439-11eb-8ff7-00344721d5cd.png)
 to 
![image](https://user-images.githubusercontent.com/4951803/104249536-a1d6a580-5439-11eb-82b7-356a4d672058.png)

Reply:
(worth noting, that to me at least, the copyright notice isn't at all important, certainly not enough for it to stick eternally to the user's window)

Reply:
Thank you for the review, looking into this issue.

Reply:
Hey, I have solved the problem by using flexbox css. I wrapped the content with a flex column container and justified content as 'space-between'. I hope this works fine now :)